devtools/server/actors/replay/module.js

```diff
@@ -1340,6 +1340,31 @@ if (isRecordingOrReplaying) {
     },
   });
 
+  Services.cpmm.addMessageListener("RecordReplaySourcemapContents", {
+    receiveMessage({ data }) {
+      RecordReplayControl.writeSourcemap(
+        data.id || "",
```

Comment:
I got a crash when one of these strings is nullish, so I replace nullish values with an empty string here. Should I handle that in the C++ code instead?

Reply:
I think it's best to pass in values of a consistent type to the C++ function, so this is fine as is.